src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  ElementRef,
+  Injectable,
+  Renderer2,
+  RendererFactory2
+} from '@angular/core';
+
+import {
+  SkyMediaBreakpoints
+} from '../media-query';
+
+/* tslint:disable */
+const SKY_TABBABLE_SELECTOR = [
+  'a[href]',
+  'area[href]',
+  'input:not([disabled]):not([tabindex=\'-1\'])',
+  'button:not([disabled]):not([tabindex=\'-1\'])',
+  'select:not([disabled]):not([tabindex=\'-1\'])',
+  'textarea:not([disabled]):not([tabindex=\'-1\'])',
+  'iframe',
+  'object',
+  'embed',
+  '*[tabindex]:not([tabindex=\'-1\'])',
+  '*[contenteditable=true]'
+].join(', ');
+/* tslint:enable */
+
+@Injectable()
+export class SkyAdapterService {
+  private renderer: Renderer2;
+
+  constructor(
+    private rendererFactory: RendererFactory2
+  ) {
+    this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
+  }
+
+  /**
+   * Set the responsive container CSS class for a given element.
+   *
+   * @param {ElementRef} elementRef - The element that will recieve the new CSS class.
+   * @param {SkyMediaBreakpoints} breakpoint - The SkyMediaBreakpoint will determine which class
+   * gets set. For example a SkyMediaBreakpoint of `xs` will set a CSS class of `sky-responsive-container-xs`.
+   */
+  public setResponsiveContainerClass(elementRef: ElementRef, breakpoint: SkyMediaBreakpoints): void {
+    const nativeEl: HTMLElement = elementRef.nativeElement;
+
+    this.renderer.removeClass(nativeEl, 'sky-responsive-container-xs');
+    this.renderer.removeClass(nativeEl, 'sky-responsive-container-sm');
+    this.renderer.removeClass(nativeEl, 'sky-responsive-container-md');
+    this.renderer.removeClass(nativeEl, 'sky-responsive-container-lg');
+
+    let newClass: string;
+
+    switch (breakpoint) {
+      case SkyMediaBreakpoints.xs: {
+        newClass = 'sky-responsive-container-xs';
+        break;
+      }
+      case SkyMediaBreakpoints.sm: {
+        newClass = 'sky-responsive-container-sm';
+        break;
+      }
+      case SkyMediaBreakpoints.md: {
+        newClass = 'sky-responsive-container-md';
+        break;
+      }
+      default: {
+        newClass = 'sky-responsive-container-lg';
+        break;
+      }
+    }
+
+    this.renderer.addClass(nativeEl, newClass);
+  }
+
+  /**
+   * This method temporarily enables/disables pointer events.
+   * This is helpful to prevent iFrames from interfering with drag events.
+   *
+   * @param {boolean} enable - Set to `true` to enable pointer events. Set to `false` to disable.
+   */
+  public toggleIframePointerEvents(enable: boolean): void {
```

Comment:
Do we do this somewhere outside of flyouts? Wondering if we need to pull this out? Either way si fine.

Reply:
Split-view will also need it, as it has dragging.